existing-solutions.tex

```diff
@@ -1,74 +1,71 @@
+nobody holds over 50\% of the whole computational power, in which case they
+could fork the chain at any point and get ahead of the main history line. This
+is a serious issue since in most protocols the most difficult chain is
+considered the proper one. Therefore one needs a lot of participants in the
+network to make it reasonably safe. Moreover, this solution leads to extreme
+waste of energy and huge costs — according to some measurements, the whole
+blockchain environment burns enough energy to power the whole
+Denmark\cite{bitcoin_energy}.
 
 This idea does not scale well — it is almost impossible to create a public
 network from scratch that would not eventually be dominated by some malicious
 entity. A lot of existing serious blockchains suffer this
-problem\cite{51attack}. On the other hand, a network becomes extremely secure once
-it is popular enough.
+problem\cite{51attack}. On the other hand, a network becomes extremely secure
+once it is popular enough.
 
 \subsection{Proof of Stake}
 
 While PoW distributes the leadership based on computational power, PoS does it
-based on so–called stake, which in most cases means token supply, sometimes
-with additional tweaks\cite{peercoin}\cite{cryptocurr_without_pow}.
-The idea is to create a leadership voting system
-which is activated periodically. Each time an election event occurs,
-the new leader is randomly selected from the stakeholders (called delegates).
-The chance of winning an election is proportional to the size of one's stake.
-This approach does
-not imply any noticeable energetic overhead and therefore is much more friendly to the
-environment. It also does not require users to have powerful computers to be able to
-have some involvement in decision making.
+based on so–called stake, which in most cases means token supply, sometimes with
+additional tweaks\cite{peercoin}\cite{cryptocurr_without_pow}. The idea is to
+create a leadership voting system which is activated periodically. Each time an
+election event occurs, the new leader is randomly selected from the stakeholders
+(called delegates). The chance of winning an election is proportional to the
+size of one's stake. This approach does not imply any noticeable energetic
+overhead and therefore is much more friendly to the environment. It also does
+not require users to have powerful computers to be able to have some involvement
+in decision making.
 
 However, PoS comes with some serious issues. First of all, there is the infamous
-"nothing at stake"\cite{pos_flaws_nothing}
-problem, which exploits the lack of any cost of the actual mining. In this case,
-there is no downside to staking several branches simultaneously in case of a fork.
+"nothing at stake"\cite{pos_flaws_nothing} problem, which exploits the lack of
+any cost of the actual mining. In this case, there is no downside to staking
+several branches simultaneously in case of a fork.
```

Comment:
More generally, the 'nothing at stake' problem can pop up when the attacker is fully willing to burn its stake.